src/client/Microsoft.Identity.Client/Internal/Logger/LoggerHelper.cs

```diff
@@ -110,6 +113,7 @@ internal static string GetPiiScrubbedExceptionDetails(Exception ex)
                 {
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "HTTP StatusCode {0}", msalServiceException.StatusCode));
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "CorrelationId {0}", msalServiceException.CorrelationId));
+                    sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "AAD Error Code AADSTS{0}", msalServiceException.ErrorCodes.FirstOrDefault()));
```

Comment:
So I see that `ErrorCodes` is an array and I am wondering if it makes sense for there to be more that one AAD error code? I've never seen it so I just picked the first one.

Reply:
This is the contract with AAD. The JSON returned is like this: 

<img width="379" alt="image" src="https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/assets/12273384/271e8aa8-ce46-4bf1-b57b-2b1af3437483">

Why pick just the first one ? How about `string.join(ErrorCodes, " ");`